extensions/wolfram-alpha/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "wolfram-alpha",
+  "title": "Wolfram Alpha",
+  "description": "Query Wolfram Alpha right from Raycast",
+  "icon": "command-icon.png",
+  "author": "mathieudutour",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Query Wolfram Alpha",
+      "description": "Template for showing a static list and a detail view",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "appId",
+      "type": "password",
+      "required": true,
+      "title": "Wolfram Alpha App ID",
+      "description": "See the extension's README for instructions about how to create an App ID",
+      "link": "https://developer.wolframalpha.com/portal/myapps/"
+    },
+    {
+      "name": "units",
+      "type": "dropdown",
+      "required": false,
+      "title": "Units",
+      "data": [
+        {
+          "title": "Metric",
+          "value": "metric"
+        },
+        {
+          "title": "Imperial",
+          "value": "imperial"
+        }
+      ],
+      "description": "The system of units to use for measurements and quantities"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.26.0",
+    "macos-audio-devices": "^1.4.0",
```

Comment:
```suggestion
```
Didn't see this the first time, should be removed

Reply:
ah yep, that's the issue when I work on too many things at the same time 😄